cli/lsp/capabilities.rs

```diff
@@ -97,7 +97,7 @@ pub fn server_capabilities(
     )),
     references_provider: Some(OneOf::Left(true)),
     document_highlight_provider: Some(OneOf::Left(true)),
-    document_symbol_provider: None,
+    document_symbol_provider: Some(OneOf::Left(true)),
```

Comment:
We should set a label here, but the lsp-types crate doesn't contain it at the moment.  I opened PR https://github.com/gluon-lang/lsp-types/pull/207 to add it, but obviously it will take a way to work through, so we should update this when the dependencies get updated.  Maybe add a TODO for now (or create a tracking issue)?

Reply:
Added a TODO comment